### ISSUE TITLE
Pathways display treatment

### ISSUE THREAD

**Issue by emgollie** [mozilla/mozbadging#46]
User feedback on recent wireframes (https://openbadges.etherpad.mozilla.org/badges-mozilla-org-wireframes) included this comment: _I wish I could see the trajectory of badges for a given contrbution pathway easily._

@zzzndr and I think that this is a nice to have and may be a post-launch project. We can decide in the coming weeks (I also need to confirm what "trajectory" means here re: related badges). cc @christensenep @andrewhayward

**Comment by emgollie** [comment]
Another related comment: _Wouldn't it be great if "favouriting" badges on badges.mozilla.org allowed you to build your own pathway at discover.openbadges.org?_

**Comment by ghost** [comment]
Sounds like this basically means a visualization of a badge pathway, if I'm guessing/understanding correctly?

**Comment by emgollie** [comment]
@christensenep That's right--an extension on some of the discovery / pathways work if you will. cc @zzzndr

**Comment by andrewhayward** [comment]
Is discover.openbadges.org actually production ready? Maybe I just misunderstood @stenington and @chloeatplay, but I was under the impression that it was kind of at demo level. Or are we just talking about doing something similar?

**Comment by chloeatplay** [comment]
@andrewhayward status of discover.openbadges.org is :point_right:  prototype.

**Comment by ghost** [comment]
Sorry, that was sort of the unspoken part of my comment.  Visualization of badge pathways, therefore post-launch.
